composer.json

```diff
@@ -13,12 +13,10 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "eloquent/pathogen": "^0.6.1"
+        "php": ">=8.1"
```

Comment:
is there anything in the code (except phpunit, but that's a dev requirement) which won't run on php 8.0?
But I get it, it's simpler to maintain when the required php version goes in line with the version required by the testing framework.

Reply:
There was a dependency (php-library) which needed it, but it was removed like yesterday. It's a matter of syntax we want to use, for example if we want to use. I think we should actually use 8.3 syntax now.

Reply:
I don't think we should force the 8.3. This library is so tiny there won't be real benefits of forcing the newer php version than necessary. Let's keep >=8.1 since that is inline with phpunit requirement and 8.1+ can be easily tested in the github workflow. (please restore 8.1 and 8.2 in tests.yml)

Reply:
This library is only used on westbahn and we plan to go straight to PHP 8.3 there. There is no real benefit to anyone to make it PHP 8.1 compatible. We should already think about 8.4 and not 8.1 @falkenhawk